### DISCUSSION TITLE
Need help getting single event page to display

### DISCUSSION THREAD

**Discussion by skennedygorges** [liedekef/events-made-easy#371] [Q&A]
I've assigned the plugin default event page in the settings and created a new page that displays the event list but when I click on an event I see the default event page and it's text instead of the event details. What am I missing? I tried creating a template and assigning it to the event and pasting in the event template code but still only seeing the default text.

**Comment by liedekef** [comment]
Check if there's no conflict in the SEO settings for the events and any page. EME will warn you if there's a problem, so if you don't see a banner mentioning that (in EME admin window), you're good to go.
Also check that your permalinks are working as expected.

**Reply by liedekef** [reply]
Next to that: mention your EME and php version.

**Comment by skennedygorges** [comment]
EME Version 2.4.16
PHP 8.3
Permalinks are working

I tried uninstalling and deleting the plugin and reinstalling but I am still seeing the default text for individual event pages.

**Comment by liedekef** [comment]
Please provide a demo then

**Comment by skennedygorges** [comment]
https://irrc.gorges.dev/events-list-page/

**Comment by liedekef** [comment]
If it doesn't work, but your permalinks work, then you should check your theme. Is it using correct get_header/get_footer/the_post/the_content calls? Some themes do things their own way, causing some wordpress filters to be bypassed ...

**Comment by skennedygorges** [comment]
I'm using the Divi theme so I would be surprised if there were issues related to that. I'm going to try again with a clean install and database and see if I still am having issues.

**Reply by liedekef** [reply]
Just try with a default WP theme (I don't think a reinstall is needed)

**Comment by skennedygorges** [comment]
Interesting. It worked with a default theme.

**Comment by liedekef** [comment]
Yeah ... I had a similar question more than 2 years ago, but never got the solution to it: 
https://wordpress.org/support/topic/divi-theme-builder-global-body-with-eme/

My best guess is that Divi is not using the standard WP calls for the_post, causing the content filter not to work as expected ... but I don't work for Divi, so that question is best asked to them (EME events are not posts, they are rendered on the special events page via a call to the_content filter).

**Comment by skennedygorges** [comment]
We can close this out. I'll go with a plugin that is compatible with Divi. Thanks for looking into it!

**Comment by liedekef** [comment]
I had several other users with Divi themes without any problem. Anyway: you're indeed free to choose any plugin of choice :-)
